commons/zenoh-config/src/wrappers.rs

```diff
@@ -99,7 +103,13 @@ impl fmt::Display for Hello {
     }
 }
 
-#[derive(Debug, Default, Copy, Clone, Eq, Hash, PartialEq)]
+impl fmt::Display for Hello {
```

Comment:
@Mallets @milyin Is there a reason to have a `Display` implementation that looks like a `Debug` one on `Hello`.
When the previous `Hello` type contained the `version` in its `Debug` implementation, why not, but now, shouldn't we remove this `Display` implementation?

Reply:
I would keep `Debug` printing all information and the `Display` a simplified version of it.

Reply:
To me the `Debug` should print the whole internal state.

Reply:
Ok, so I will fix the `Debug` impl.